src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.models.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+    public Book findByAuthor(String author);
```

Comment:
This method should return `Optional<Book>`, that way you won't have to check for null values while using it in the future 😄  
https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html